modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotStorage.scala

```diff
@@ -60,6 +65,40 @@ object GlobalSnapshotStorage {
       }
   }
 
+  def make[F[_]: KryoSerializer](
+    ref: MapRef[F, SnapshotOrdinal, Option[GlobalSnapshot]]
+  )(implicit F: Async[F]): GlobalSnapshotStorage[F] =
+    new GlobalSnapshotStorage[F] {
+
+      implicit val ordinalOrder: Order[SnapshotOrdinal] = Order[Long].contramap[SnapshotOrdinal](_.value.value)
```

Comment:
Can we put that to companion object of `SnapshotOrdinal`?

Reply:
We decided to use order/ordering locally

Reply:
My understanding was that we agreed to use order/ordering locally only if it doesn't include all the fields.

`SnapshotOrdinal` companion object already has an order instance that takes into account all of its fields (which in this case is just a single field).

Reply:
Yeah. SnapshotOrdinal is a newtype for Long so we can move it next to the type, as there will be only one ordering. For other data types I'd use ordering locally until used twice

Reply:
I think that the rule of using order/ordering locally only if it doesn't include all the fields makes more sense than the rule of using it locally only if it's used once. It's hard to keep track of how many orderings for a given type are there in the whole codebase.